### ISSUE TITLE
Missing .conf extension for sites-available/* sites-enabled/*

### ISSUE THREAD

**Issue by hrak** [svanzoest/apache2-cookbook#315]
In this bit of code, the cookbook wants to delete the default vhost config installed by the apache2 package
```ruby
%w(default 000-default).each do |site|
  link "#{node['apache']['dir']}/sites-enabled/#{site}" do
    action :delete
  end

  file "#{node['apache']['dir']}/sites-available/#{site}" do
    action :delete
    backup false
  end
end
```
On Ubuntu 14.04 and Debian Sid, the files in sites-available/ and the links in sites-enabled/ have a .conf extension ('000-default.conf') so they are never removed.

**Comment by svanzoest** [comment]
Thanks @hrak 
There are a few different platforms, some with `.conf`, some without and I can easily see how we may have missed this.

**Comment by hrak** [comment]
#188 seems related, and also mentions 000-default.conf, but the fix applied in that bug report still does not cover the .conf extension. The only thing that sort of fixed #188 was this change:

```ruby
+apache_site '000-default' do
+ enable node['apache']['default_site_enabled']
+end
```
that re-enables the never removed 000-default.conf (ignoring the 'sites-available/default.conf' that gets installed by the cookbook). This bit is now removed from the current version of the cookbook.

**Comment by svanzoest** [comment]
I think my recent commit resolves this, can you confirm?
